packages/ariakit/src/combobox/combobox-item.tsx

```diff
@@ -1,10 +1,19 @@
-import { KeyboardEvent, MouseEvent, useCallback } from "react";
+import {
+  KeyboardEvent,
+  MouseEvent,
+  MouseEvent as ReactMouseEvent,
```

Comment:
We're already importing `MouseEvent` from React here, so there is no need to import it again. We usually rename it to `ReactMouseEvent` when using the DOM `MouseEvent` type in the file.
```suggestion
  MouseEvent,
```

Reply:
Ah, gotcha :+1: